lib/index.js

```diff
@@ -77,11 +76,10 @@ class PWMetrics {
     return lighthouse(this.url, lhOpts, perfConfig)
       .then(res => metrics.prepareData(res))
       .then(data => {
-        if (this.flags.submit) {
-          return this.appendResults(data);
```

Comment:
I had put it here so I would get results streaming into sheets continuously if i did --runs=100. With this move, it looks like it'd only report them all when the 100 finish, am i reading this right?